core/src/system_monitor_service.rs

```diff
@@ -387,21 +385,25 @@ fn parse_disk_stats(reader_diskstats: &mut impl BufRead) -> Result<DiskStats, St
     Ok(stats)
 }
 
-pub struct SystemMonitorStatsReportConfig {
```

Comment:
mind popping the config struct out and re-introducing after merging this PR?

Reply:
sure.

Reply:
https://github.com/solana-labs/solana/pull/29132